### PR TITLE
Fix transform on clipped images

### PR DESCRIPTION
**Pull request by stephsharp**
transform-fix

**Note:** depends on #436

### CODE REVIEW

**Comment by stephsharp** [comment]
Investigating the broken test

**Comment by f3dm76** [comment]
Hello, Thank you very much for your contribution. About the test, you are right. I added serialization for `TransformedLocus`. We'll let you know once it is merged

File: Source/svg/SVGParser.swift [hunk 1]
```diff
@@ -1393,6 +1393,9 @@ open class SVGParser {
                     let transform = transformBoundingBoxLocus(respectiveLocus: userSpaceLocus.locus, absoluteLocus: locus)
                     return TransformedLocus(locus: userSpaceLocus.locus, transform: transform)
                 }
+                if let userSpaceTransform = userSpaceLocus.transform, userSpaceTransform != Transform.identity {
```
Comment (stephsharp) [inline Source/svg/SVGParser.swift]: I added a check here to only return a `TransformedLocus` if the transform is not the identity matrix.

**Comment by stephsharp** [comment]
Thanks @f3dm76, tests are passing now!

**Comment by f3dm76** [comment]
Hello, looks good, but it seems like it's easier to just put `TransformedLocus` directly into `UserSpaceLocus.locus`. Then there's no need for `UserSpaceLocus.locus` at all.

**Comment by ystrot** [comment]
Hi Stephanie,

Thanks for your help!

**Pull request merged**